src/EnergyPlus/GroundTemperatureModeling/XingGroundTemperatureModel.cc

```diff
@@ -116,7 +109,7 @@ std::shared_ptr<XingGroundTempsModel> XingGroundTempsModel::XingGTMFactory(Energ
         }
     }
 
-    if (found && !ErrorsFound) {
+    if (found) {
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/XingGroundTemperatureModel.cc:119]:(style),[knownConditionTrueFalse],Condition '!ErrorsFound' is always true